toolchain/semantics/semantics_ir.cpp

```diff
@@ -241,6 +242,27 @@ auto SemanticsIR::StringifyType(SemanticsTypeId type_id) -> std::string {
         steps.push_back({.node_id = GetTypeAllowBuiltinTypes(node.type_id())});
         break;
       }
+      case SemanticsNodeKind::TupleType: {
+        auto refs = GetTypeBlock(node.GetAsTupleType());
+        if (refs.empty()) {
+          out << "() as type";
+          break;
+        } else if (step.index == 0) {
+          out << "(";
+        } else if (step.index < static_cast<int>(refs.size())) {
+          out << ", ";
+        } else {
+          if (step.index == 1) {
```

Comment:
```suggestion
          // A tuple of one element has a comma to disambiguate from an expression.
          if (step.index == 1) {
```

Let's comment this behavior.